README.md

```diff
@@ -1,2 +1,24 @@
 # QuDynamics.jl
+
 A JuliaQuantum package for solving dynamical equations in quantum mechanics.
+
+The package aims to provide a flexible and unified framework for solving dynamical equations in quantum mechanics. The package will make use of existing packages mainly QuBase.jl in addition to various other solver packages like ODE.jl, Expokit.jl.
+Some of the discussion regarding design and implementation can be found [here](https://github.com/JuliaQuantum/JuliaQuantum.github.io/issues/20) and [here](https://github.com/numfocus/gsoc/blob/7917b4fc08ce73ac74f8a2b9dd7929d994fc4282/2015/proposals/Amit.md). Updates of the implemented features can be followed on the [news page](http://juliaquantum.github.io/news/) of the JuliaQuantum website.
+
+**Dependencies**
+QuBase.jl
+
+This has to be added using
+```julia
+julia>Pkg.clone([https://github.com/JuliaQuantum/QuBase.jl](https://github.com/JuliaQuantum/QuBase.jl))
```

Comment:
Ah! I was trying to provide the link which seems to be wrong. Anyways edits done. Sorry for that I overlooked it.